api/python/provisioner/api.py

```diff
@@ -578,17 +578,20 @@ def replace_node(node_id, node_host=None, node_port=None, nowait=True):
     )
 
 
-def create_user(uname, passwd, targets=ALL_MINIONS, nowait=False):
+def create_user(uname, passwd, sudo=True, group_list=[], commands_list=[], targets=ALL_MINIONS, nowait=False):
```

Comment:
Python has a better way for declaring types now:
```
def create_user(
    uname: str,
    passwd: str,
    sudo=True: bool,
    group_list: list,
    commands_list: list,
    targets=ALL_MINIONS,
    nowait=False: bool
):
```

Reply:
Addressed